index.md

```diff
@@ -1,9 +1,80 @@
+The overall philosophy for builds and deployment and the development process can be found in {cite:p}`2018SPIE10707E..09J` and the [Rubin developer guide](https://developer.lsst.io).
+
+## Responsibilities
+
+The role of the Build Engineering Team is to facilitate releases of the LSST Science Pipelines software to the community and other parts of the Rubin organization.
+This includes the support and evolutions of:
+
+* Making formal science pipelines releases (working with the pipelines team on release notes and timing as well as the infrastructure for making the releases).
+* The Conda python environment (e.g., the `rubin-env` meta package)
+* The pipeline build system used by the developers (`sconsUtils`, EUPS, lint tooling, `lsstsw`, `lsstinstall`)
+* The GitHub Actions and associated template packages for pipelines packages.
+* The Python/C++ bindings such as Pybind11 or Nanobind.
+* The CI/CD infrastructure (nightly, weekly, ticket branch releases, making docker containers, running AP verification jobs) on multiple architectures.
+
+A detailed overview of the build system can be found in {cite:p}`DMTN-218`.
+
+The BET can investigate new technologies for building or deploying the software such as:
+
+* Changing from Docker containers to a new technology as driven by stakeholders.
+* Using a completely different CI/CD system.
+* Having alternate distributions mechanisms that do not use EUPS (such as Conda).
+* Replacing SCons as the package build system.
+
+As we head into the era of long-lived formal data releases there is an expectation that BET would be involved in any changes to how we coordinate branches and release strategies in Git.
+
+## Stakeholders
+
+The Build Engineering Team (BET) is a small but critical team that sits in the middle of many stakeholders.
+The Science Pipeline team relies on BET to ensure that not only releases are made but also to ensure that they can build and deploy the software themselves as well as run it during data release processing.
+
+The Telescope and Site software team use the Science Pipeline container as the core of the CSC containers that they deploy to the summit.
+Any changes to Python major versions must be approved and coordinated by T&S as well as with the Science Pipelines team.
+
+The Rubin Science Platform deploys the pipeline docker containers.
+These containers form the primary interface used by the science community.
+The RSP community, including the Community Science Team, expects to have access to containers from weekly releases, formal validated releases, and recent dailies.
+
+Finally, science users who are not using the containers need to be able to install the software.
+The requirement is that the software be installable to outside users but this system can change over time.
+
```

Comment:
Might list DM-CCB here as a stakeholder in the sense of relying on the release process as well as relying on BET expertise to inform decisions.